docs/pages/kmodal.vue

```diff
@@ -11,6 +11,54 @@
         text="modals"
       />.
     </DocsPageSection>
+    <DocsPageSection
+      title="Usage"
+      anchor="#usage"
+    >
+      <DocsSubNav
+        :items="[
+          { text: 'Dialog box with description', href: '#dialog-box' },
+          { text: 'Submit form dialog box', href: '#submit-form' },
+          { text: 'Modal with different sizes', href: '#modal-sizes' },
+        ]"
+      />
+
+      <h3>
+        Dialog box with description
+        <DocsAnchorTarget anchor="#dialog-box" />
+      </h3>
+      <p>To show message information, warnings, or notifications to the user</p>
```

Comment:
```suggestion
      <p>To show information, warnings, or notifications.</p>
```

For conciseness. Also adding dot.